test/run_pass/test_fs_existssync.js

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2016 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var fs = require('fs');
+var assert = require('assert');
+
+{
+  var filePath = "../resources/tobeornottobe.txt";
+
+  var result = fs.existsSync(filePath);
+  assert.equal(result, true);
+}
+
+{
+  var filePath = "../resources/empty.txt";
+
+  var result = fs.existsSync(filePath);
+  assert.equal(result, false);
+}
+
+{
+  var filePath = "";
+
+  var result = fs.existsSync(filePath);
+  assert.equal(result, false);
+}
+
+{
+  var filePath = " ";
```

Comment:
Just a note, ECMAScript has only closure scope, so all of these filePath variables are the same variable.